scripts/linters/codeowner_linter_test.py

```diff
@@ -89,28 +89,30 @@ def test_missing_important_codeowner_path_from_list(self):
             codeowner_linter, 'CODEOWNER_FILEPATH',
             VALID_CODEOWNER_FILEPATH)
 
-        with self.print_swap, self.listdir_swap, codeowner_important_paths_swap:
+        with self.listdir_swap, codeowner_important_paths_swap:
             with codeowner_filepath_swap:
-                codeowner_linter.check_codeowner_file(FILE_CACHE, True)
+                linter = codeowner_linter.CodeOwnerLintChecksManager(FILE_CACHE)
+                linter_stdout = linter.check_codeowner_file()
         self.assert_same_list_elements([
```

Comment:
Hi @DubeySandeep, I don't think this was supposed to be done in this PR. It will be done in separate PR where we will match the complete logs. Refer: https://github.com/oppia/oppia/pull/9925#discussion_r458912925

Reply:
We are changing the private linter functions to the public so that we can also write unit tests for each function and assert the complete error log reported by the function, that was the expectation, right? Are we thinking of doing this in a separate PR? I'm fine with that but I think the main expectations from these public functions are that we can run them separately and can have test complete logs and remove `assert_same_list_elements` function from the codebase completely!

Reply:
Yes, the expectation is totally correct.

Yup, we plan to do it in a separate PR.

Reply:
> Yup, we plan to do it in a separate PR.

I'm fine with that, but please make sure to remove the `assert_same_list_elements` from the codebase before we start using it in other places! [It would be great if you can prioritize that PR, I think that was the major expectation from this PR.]

Reply:
Sure.